src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
 List of common options:
 
-  --printconf          Print the location of the default configuration file.
+  --printconf              Print the location of the default configuration file.
```

Comment:
Change the above to "Available options" or something similar (we are aiming to list everything here).

Reply:
About `--printconf`, hold on to your hat! This also _writes_ the default config if it does not exist 😐 Well, that's how it works, so we need to document it...

Reply:
Changed to "available"

Reply:
> About `--printconf`, hold on to your hat! This also _writes_ the default config if it does not exist neutral_face Well, that's how it works, so we need to document it...

So it does.  That's unexpected.  Documented.